fe/src/app/globals.css

```diff
@@ -1,3 +1,10 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+
+.date-color-override {
```

Comment:
was this needed because tailwind styles didn't work?

Reply:
Added custom color to tailwind theme.